tests/baselines/reference/jsDeclarationsPrivateFields01.js

```diff
@@ -0,0 +1,30 @@
+//// [file.js]
+export class C {
+    #hello = "hello";
+    #world = 100;
+
+    #calcHello() {
+        return this.#hello;
+    }
+
+    get #screamingHello() {
```

Comment:
I find this really funny, but can't calc why

Reply:
This is, for some reason, the only example code I can think of when I try to write computed properties.

Reply:
This is, for some reason, the only example code I can think of when I try to write computed properties.